### PR TITLE
CI: build llm-load-tests on every merge

### PR DESCRIPTION
**Pull request by ccamacho**
This commit builds a container image with each
code change.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        uses: actions/checkout@v3
+      - name: Buildah build
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: ${{ github.event.repository.name }}
+          tags: latest ${{ github.sha }}
+          containerfiles: |
+            ./build/Containerfile
+      - name: Push To ghcr.io
```
Comment (sjmonson) [inline .github/workflows/build.yml:29]: We should probably be using quay.io rather then Github's image registry.

Comment (ccamacho) [reply .github/workflows/build.yml:29]: The thing is that the ghcr releases will be part of the same repo, as a release, same inteface, everying integrated. We wont need to ask for api keys that might expire.. @dagrayvid @kpouget @npalaska open to both options not a problem.

Comment (dagrayvid) [reply .github/workflows/build.yml:29]: I am fine with using ghcr.io for the sake of simplicity.

Comment (sjmonson) [reply .github/workflows/build.yml:29]: >  We wont need to ask for api keys that might expire..

There are no api keys involved in the quay approach afaik. It just pulls the repo, if the repo is private we would have to add a deploy key but that doesn't expire.

However if everyone is in agreement on ghcr I am fine with it. I just want to share the same process across all our tools.

**Review by sjmonson — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (dagrayvid) [reply .github/workflows/build.yml:29]: Creds with write permission would be needed to push an image to the quay repo, I think this is what Carlos was referring to?

Comment (sjmonson) [reply .github/workflows/build.yml:29]: Yes if we were manually pushing images or using an external build system a key would be necessary. However, I was referring to using a quay build trigger which is a pull-based system.

Comment (dagrayvid) [reply .github/workflows/build.yml:29]: Ah, thanks for clarifying. In that case, I don't see a clear advantage to either.

**Pull request merged**